core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -199,6 +199,18 @@ private[spark] object Utils extends Logging with SparkClassUtils {
     Try { classForName(clazz, initialize = false) }.isSuccess
   }
 
+  def getQuantilesValue(
```

Comment:
I don't see a good reason to put this in Utils; it's not used more widely?

Reply:
I was indifferent to putting it in Utils but since it is currently used in two different classes, decided to put it there.

Reply:
Can it go somewhere 'closer' to the places it is used? it may not be worth refactoring

Reply:
Moved it to `AppStatusUtils`, thanks.